Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -49,8 +55,16 @@ def PyInit(self):
         self.declareProperty(name='CanShiftFactor', defaultValue=0.0,
                              doc='Amount by which to shift the container data')
 
-        self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '', direction=Direction.Output),
+        self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '',
+                             direction=Direction.Output),
                              doc='The output corrections workspace.')
+        self.declareProperty(name='RebinCanToSample',
+                             defaultValue='Rebin Can',
+                             validator=StringListValidator([
+                                 'Rebin Can',
+                                 'Do Not Rebin Can']),
+                             doc=('Enable or disable RebinToWorkspace on ' +
+                                  'CanWorkspace.'))
```

Comment:
When calling algorithms there is a tendency to use 
`ApplyPaalmanPingsCorrection(SampleWorkspace=sample_1, CanWorkspace=container_1, RebinCanToSample='True')`
so the code is still self-documented
It also means you can use get rid of:
```
if self.getProperty('RebinCanToSample').value == 'Rebin Can':
    self._rebin_container_ws = True
else:
    self._rebin_container_ws = False
```